ycms/cms/templates/_base.html

```diff
@@ -18,22 +18,32 @@
 limitations under the License.
 {% endcomment %}
 {% block raw_content %}
-    <div class="lg:flex">
+    <div class="lg:flex dark:bg-gray-900">
```

Comment:
```suggestion
    <div class="lg:flex dark:bg-gray-900 min-h-full">
```
Currently, if the content isn't taking up the full page, the white background is showing